contracts/interfaces/IDnGmxJuniorVault.sol

```diff
@@ -94,6 +94,8 @@ interface IDnGmxJuniorVault is IERC4626, IBorrower {
 
     event TraderOIHedgesUpdated(int256 btcTraderOIHedge, int256 ethTraderOIHedge);
 
+    event AssetSlippage(uint slippage);
```

Comment:
we should add more one more indexed field for uniqueness to differentiate b/w deposit, withdraw, mint or redeem. for e.g `msg.sig`